autogpt/commands/image_gen.py

```diff
@@ -22,7 +22,11 @@ def generate_image(prompt: str) -> str:
     Returns:
         str: The filename of the image
     """
-    filename = f"{str(uuid.uuid4())}.jpg"
+    ts =  time.time()
+    """First 3 words for the file name"""
+    split = prompt.split()[0:3]
+    name  = "-".join(split)
+    filename = name + str(ts) + ".jpg"
```

Comment:
Why don’t you let the model decide the filename instead of choosing it from the prompt which used to generate it? 

Add <filename> to the prompt and you will end up with more quality names.

Reply:
For now, I would feel this would work fine. It's done well

Reply:
It's better than the way it was before

Reply:
Should have an underscore between the time and name though.